didx-xyz/yoma-mobile
# YOMA-457 - feat(Completed Challenges): view with real data

## Scope [YOMA-457](https://yomaworld.atlassian.net/browse/YOMA-457)
- adding real data to the Completed Challenges view

![](https://media.giphy.com/media/5ObsvjbPzTm9y/giphy.gif)
## Work Done
- adding selector and container and utils for Completed Challenges view to bring in the correct data. 

## Steps to Test
run tests
open Completed Challenges widget (on MyCV) > View All - see the data loads correctly. 
    
## Screenshots
<img width="425" alt="Screenshot 2021-09-08 at 00 10 05" src="https://user-images.githubusercontent.com/1424561/132417144-e60d05b0-82b3-45c8-8735-b96162b3541a.png">
